docs/development/releasing.rst

```diff
@@ -228,6 +234,11 @@ packages that use the full bugfix/maintenance branch approach.)
       Also, it might be tempting to use the ``--tags`` argument to ``git push``,
       but this should *not* be done, as it might push up some unintended tags.
 
+.. _post-release-procedure:
+
+Post-Release procedures
```

Comment:
```suggestion
Post-Release Procedures
```

Reply:
Not sure about this one, as the other existing headings only capitalized the first letter of the first word.

Reply:
No worries, will edit all of the headings when I get to this sub-package!